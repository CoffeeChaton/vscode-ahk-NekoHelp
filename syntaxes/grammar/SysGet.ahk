SysGet, OutputVar, MonitorCount
SysGet, OutputVar, MonitorPrimary
SysGet, OutputVar, Monitor , N
SysGet, OutputVar, MonitorWorkArea , N
SysGet, OutputVar, MonitorName , N
SysGet, OutputVar, 65

SysGet OutputVar, MonitorCount
SysGet OutputVar, MonitorPrimary
SysGet OutputVar, Monitor , N
SysGet OutputVar, MonitorWorkArea , N
SysGet OutputVar, MonitorName , N
SysGet OutputVar, 65