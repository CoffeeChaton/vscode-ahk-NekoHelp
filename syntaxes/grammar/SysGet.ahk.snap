>SysGet, OutputVar, MonitorCount
#^^^^^^ source.ahk meta.command.line.sysget.ahk support.function.command.sysget.ahk
#      ^^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk
#                   ^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk keyword.command.option.param.sysget.ahk
>SysGet, OutputVar, MonitorPrimary
#^^^^^^ source.ahk meta.command.line.sysget.ahk support.function.command.sysget.ahk
#      ^^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk
#                   ^^^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk keyword.command.option.param.sysget.ahk
>SysGet, OutputVar, Monitor , N
#^^^^^^ source.ahk meta.command.line.sysget.ahk support.function.command.sysget.ahk
#      ^^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk
#                   ^^^^^^^ source.ahk meta.command.line.sysget.ahk keyword.command.option.param.sysget.ahk
#                          ^ source.ahk meta.command.line.sysget.ahk
#                           ^ source.ahk meta.command.line.sysget.ahk punctuation.ahk
#                            ^^ source.ahk meta.command.line.sysget.ahk
>SysGet, OutputVar, MonitorWorkArea , N
#^^^^^^ source.ahk meta.command.line.sysget.ahk support.function.command.sysget.ahk
#      ^^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk
#                   ^^^^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk keyword.command.option.param.sysget.ahk
#                                  ^ source.ahk meta.command.line.sysget.ahk
#                                   ^ source.ahk meta.command.line.sysget.ahk punctuation.ahk
#                                    ^^ source.ahk meta.command.line.sysget.ahk
>SysGet, OutputVar, MonitorName , N
#^^^^^^ source.ahk meta.command.line.sysget.ahk support.function.command.sysget.ahk
#      ^^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk
#                   ^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk keyword.command.option.param.sysget.ahk
#                              ^ source.ahk meta.command.line.sysget.ahk
#                               ^ source.ahk meta.command.line.sysget.ahk punctuation.ahk
#                                ^^ source.ahk meta.command.line.sysget.ahk
>SysGet, OutputVar, 65
#^^^^^^ source.ahk meta.command.line.ahk support.function.ahk
#      ^ source.ahk meta.command.line.ahk punctuation.ahk
#       ^^^^^^^^^^ source.ahk meta.command.line.ahk
#                 ^ source.ahk meta.command.line.ahk punctuation.ahk
#                  ^ source.ahk meta.command.line.ahk
#                   ^^ source.ahk meta.command.line.ahk constant.numeric.decimal.ahk
>
>SysGet OutputVar, MonitorCount
#^^^^^^ source.ahk meta.command.line.sysget.ahk support.function.command.sysget.ahk
#      ^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk
#                  ^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk keyword.command.option.param.sysget.ahk
>SysGet OutputVar, MonitorPrimary
#^^^^^^ source.ahk meta.command.line.sysget.ahk support.function.command.sysget.ahk
#      ^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk
#                  ^^^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk keyword.command.option.param.sysget.ahk
>SysGet OutputVar, Monitor , N
#^^^^^^ source.ahk meta.command.line.sysget.ahk support.function.command.sysget.ahk
#      ^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk
#                  ^^^^^^^ source.ahk meta.command.line.sysget.ahk keyword.command.option.param.sysget.ahk
#                         ^ source.ahk meta.command.line.sysget.ahk
#                          ^ source.ahk meta.command.line.sysget.ahk punctuation.ahk
#                           ^^ source.ahk meta.command.line.sysget.ahk
>SysGet OutputVar, MonitorWorkArea , N
#^^^^^^ source.ahk meta.command.line.sysget.ahk support.function.command.sysget.ahk
#      ^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk
#                  ^^^^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk keyword.command.option.param.sysget.ahk
#                                 ^ source.ahk meta.command.line.sysget.ahk
#                                  ^ source.ahk meta.command.line.sysget.ahk punctuation.ahk
#                                   ^^ source.ahk meta.command.line.sysget.ahk
>SysGet OutputVar, MonitorName , N
#^^^^^^ source.ahk meta.command.line.sysget.ahk support.function.command.sysget.ahk
#      ^^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk
#                  ^^^^^^^^^^^ source.ahk meta.command.line.sysget.ahk keyword.command.option.param.sysget.ahk
#                             ^ source.ahk meta.command.line.sysget.ahk
#                              ^ source.ahk meta.command.line.sysget.ahk punctuation.ahk
#                               ^^ source.ahk meta.command.line.sysget.ahk
>SysGet OutputVar, 65
#^^^^^^ source.ahk meta.command.line.ahk support.function.ahk
#      ^^^^^^^^^^ source.ahk meta.command.line.ahk
#                ^ source.ahk meta.command.line.ahk punctuation.ahk
#                 ^ source.ahk meta.command.line.ahk
#                  ^^ source.ahk meta.command.line.ahk constant.numeric.decimal.ahk